dartsim/src/Base.hh

```diff
@@ -291,6 +293,15 @@ class Base : public Implements3d<FeatureList<Feature>>
     return std::forward_as_tuple(id, entry);
   }
 
+  public: inline std::tuple<std::size_t, ModelInfo &> AddNestedModel(
+              const ModelInfo &_info, const std::size_t _parentID,
+              const std::size_t _worldID)
```

Comment:
If I recall correctly, according to the Ignition style guide, we don't mark method parameters with primitive data types as `const`. I'm not sure if `std::size_t` counts towards this criteria or not.

Reply:
When `const` is used for the parameter type of a function *definition* then it serves an important role in const-correctness, ensuring that the parameter's value is never modified anywhere inside the function definition.

The style guide probably says that the function *declaration* should not use `const` on primitive-type parameters, which is fine because whether or not you put `const` on pass-by-value parameter does not affect the function signature.

You can see more about this topic in [this discussion](https://stackoverflow.com/questions/117293/use-of-const-for-function-parameters).

Reply:
Ah, yes, that's correct, I forgot about the nuances between definitions and declarations. This looks good to me then, so I'll mark this as resolved.

Reply:
Thanks for the explanation @mxgrey!